store/modules/actions/payment.js

```diff
@@ -41,25 +44,51 @@ export async function sendCosmosPayment(
         values,
         memo,
       }, signer));
+    } else if (!isWordPressSideBar) {
+      ({ txHash, included } = await transferCosmos(
+        {
+          from,
+          to,
+          value,
+          memo,
+        },
+        signer,
+      ));
     } else {
-      ({ txHash, included } = await transferCosmos({
-        from,
-        to,
-        value,
-        memo,
-      }, signer));
+      const txRaw = await signCosmos(
+        {
+          from,
+          to,
+          value,
+          memo,
+        },
+        signer,
+      );
+      if (txRaw) {
+        commit(types.UI_SET_SIGN_FINISH);
+        commit(types.UI_SET_TX_SUCCESS);
```

Comment:
Refactored to only `UI_SET_TX_FAILED`, deleted `UI_SET_TX_SUCCESS` as it may mislead.
Will further refactor them for non-wordpress flow later on in mixin